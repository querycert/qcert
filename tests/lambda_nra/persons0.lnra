/* Example returning one of the inputs */

Persons

