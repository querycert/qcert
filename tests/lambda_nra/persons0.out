[{"age": {"$nat":32}, "company": {"$nat":101}, "name": "John Doe", "pid": {"$nat":1}, "salary": 1000.0}, {"age": {"$nat":32}, "company": {"$nat":103}, "name": "Jane Doe", "pid": {"$nat":2}, "salary": 1500.0}, {"age": {"$nat":34}, "company": {"$nat":101}, "name": "Jim Does", "pid": {"$nat":3}, "salary": 800.0}, {"age": {"$nat":32}, "company": {"$nat":102}, "name": "Jill Does", "pid": {"$nat":4}, "salary": 1000.0}, {"age": {"$nat":34}, "company": {"$nat":101}, "name": "Joan Doe", "pid": {"$nat":5}, "salary": 900.0}, {"age": {"$nat":35}, "company": {"$nat":103}, "name": "James Do", "pid": {"$nat":6}, "salary": 1200.0}]